src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+---
+title: 'SCIM API tutorial'
+tags:
+  - Accounts
+  - Accounts and billing
+  - Automated user management
+  - New Relic One user management
+metaDescription: 'New Relic SCIM API tutorial: automated user management of groups and users using the SCIM API.'
+---
+
+This tutorial will walk you through some common procedures for managing users on the New Relic One user model using the SCIM API. The SCIM API allows you to view, create, update, and delete users and groups programmatically, outside of the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where).
+
+## Requirements [#requirements]
+
+Before starting this tutorial, please read the [requirements for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign) and for [the SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management). 
+
+Note that the API does **not** give you the ability to manage access grants (assigning your New Relic users access to accounts and roles). This means that after you're done creating an integration with this API, you must next use the **Organization and access** UI to create access grants. For more information, see [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
+
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [Our primary SCIM API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/)
+* Some SCIM 2.0 RFC documents from the Internet Engineering Task Force that are most relevant: [RFC 7643 - SCIM Core Resources and Extensions](https://datatracker.ietf.org/doc/html/rfc7643#section-4), [RFC 7643 - JSON Representation](https://datatracker.ietf.org/doc/html/rfc7643#section-8), and [RFC 7644 - SCIM Protocol](https://datatracker.ietf.org/doc/html/rfc7644).
+
+This tutorial walks you through how to:
```

Comment:
I find this bulleted list pretty hard to parse, actually. I'm questioning how useful it is. Also, this bulleted list is replicated in the right sidebar so it's a little redundant too.

Maybe instead of an overview list, it would be more helpful to describe what someone will be able to do after completing this tutorial? Like, what's the value in doing this?

Reply:
Also, some of these high-level procedures seem required (Configure your authentication domain) while some feel potentially optional (Delete users and groups). Maybe it's helpful to break these up and into stuff that you need to know how to do and stuff that's nice to know how to do?

In other words, adding some additional organization to this doc might really help people make sense of it.